lib/msal-common/src/request/BaseAuthRequest.ts

```diff
@@ -30,5 +31,6 @@ export type BaseAuthRequest = {
     resourceRequestMethod?: string;
     resourceRequestUri?: string;
     sshJwk?: string,
-    sshKid?: string
+    sshKid?: string,
+    requestedClaimsHash?: string;
```

Comment:
Would applications be expected to pass this in? If so, can you please provide an example of what that would look like?

Reply:
No, this is calculated by MSAL based on the `claims` value. I just added it here so it can be set and retrieved later on. Is this the right place for it?

Reply:
I would prefer not to expose it in the API if the it is not intended to be something the application can set.